pom.xml

```diff
@@ -231,7 +231,13 @@
     <dependency>
       <groupId>pentaho</groupId>
       <artifactId>guava-cache-provider</artifactId>
-      <version>7.1-SNAPSHOT</version>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pdi-engine-api</artifactId>
```

Comment:
Tests were failing because of java.lang.NoClassDefFoundError: org/pentaho/di/engine/api/Engine and pdi-engine-api is where the class is. Is there a better candidate?